src/main/java/org/ilgcc/app/inputs/Gcc.java

```diff
@@ -66,7 +66,12 @@ public class Gcc extends FlowInputs {
 
     private List<String> weeklySchedule;
 
-
+    private String activitiesProgramStartDateDay;
+    private String activitiesProgramStartDateMonth;
+    private String activitiesProgramStartDateYear;
+    private String activitiesProgramEndDateDay;
+    private String activitiesProgramEndDateMonth;
+    private String activitiesProgramEndDateYear;
```

Comment:
Non-blocking comment: I think have `Date` is redundant. But that's a personal preference.

```suggestion
    private String activitiesProgramStartDay;
    private String activitiesProgramStartMonth;
    private String activitiesProgramStartYear;
    private String activitiesProgramEndDay;
    private String activitiesProgramEndMonth;
    private String activitiesProgramEndYear;
```